presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -279,6 +279,14 @@ Calculating the approximate distinct count can be done much more cheaply than an
     A P4HyperLogLog sketch is similar to :ref:`hyperloglog_type`, but it starts (and remains)
     in the dense representation.
 
+KHyperLogLog
+------------
+
+KHyperLogLog is a data sketch that can be used to compactly represents the association of two
+columns. See :doc:`/functions/khyperloglog`.
+
+.. _khyperloglog_type:
```

Comment:
I believe you need to add the label before the `KHperLogLog` heading. See example right above.

Reply:
It seems redundant to add the section header plus a sub-header with the exact same name.  Nevertheless, changed it to that for consistency.

Reply:
Looks like we have an misunderstanding here. We don't need the sub-header. I mean to put line 291 before line 285. Line 291 is a label, which is used by other parts of the doc to reference the type. Placing the reference here is essentially making a link to the next section "Quantile Digest", instead of to the `KHyperLogLog` section.

Reply:
Just to avoid confusion, by line 291, I meant this line:
```
.. _khyperloglog_type:
```